api/app/models/land_change/natural_lands.py

```diff
@@ -0,0 +1,56 @@
+from typing import Annotated, Optional, Union
+
+from pydantic import Field
+
+from ..common.analysis import AnalysisStatus
+from ..common.areas_of_interest import (
+    AdminAreaOfInterest,
+    CustomAreaOfInterest,
+    IndigenousAreaOfInterest,
+    KeyBiodiversityAreaOfInterest,
+    ProtectedAreaOfInterest,
+)
+from ..common.base import Response, StrictBaseModel
+
+AoiUnion = Union[
+    AdminAreaOfInterest,
+    KeyBiodiversityAreaOfInterest,
+    ProtectedAreaOfInterest,
+    IndigenousAreaOfInterest,
+    CustomAreaOfInterest,
+]
+
+
+class NaturalLandsAnalyticsIn(StrictBaseModel):
+    aoi: Annotated[AoiUnion, Field(discriminator="type")] = Field(
+        ...,
+        title="AOI",
+        description="AOI to calculate in.",
+    )
+
+
+class NaturalLandsAnalytics(StrictBaseModel):
+    result: Optional[dict] = {  # column oriented for loading into a dataframe
+        "__dtypes__": {
+            "country": "str",
+            "region": "int",
+            "subregion": "int",
+            "natural_lands_area": "float32",
+        },
+        "country": ["BRA", "BRA", "BRA"],
+        "region": [1, 1, 1],
+        "subregion": [12, 12, 12],
+        "natural_lands_area": [4851452000, 4084129000, 253008000],
+    }
```

Comment:
I just posted about this in @solomon-negusse 's PR review. Here's a better way to provide these hints to Swagger:
https://github.com/wri/project-zeno-data-infra/pull/47#discussion_r2248210469